paywall/src/__tests__/unlock.js/CheckoutUIHandler.test.ts

```diff
@@ -0,0 +1,193 @@
+import FakeWindow from '../test-helpers/fakeWindowHelpers'
+import CheckoutUIHandler from '../../unlock.js/CheckoutUIHandler'
+import { PaywallConfig, Locks } from '../../unlockTypes'
+import IframeHandler from '../../unlock.js/IframeHandler'
+import { PostMessages, ExtractPayload } from '../../messageTypes'
+
+declare const process: {
+  env: {
+    PAYWALL_URL: 'http://paywall'
+    USER_IFRAME_URL: 'http://app/account'
+  }
+}
+
+describe('CheckoutUIHandler', () => {
+  process.env.PAYWALL_URL = 'http://paywall'
```

Comment:
I have seen this pattern a couple times and I think this is not the right way of doing this and will come back and bite us. Let's add a note to never use (and especially overwrite) `process.env` directly in the application code.
We should map the values of `process.env` inside a config file and then import ther confg file where we need it.

Reply:
sure. The data iframe does this, and it is a pretty trivial change with a big payoff. I'll do that as a follow-up PR and modify the ones that are already out there too